src/deepsparse/yolact/utils/annotate.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
Very similar to the analogous code for yolo, probably a good idea to merge them.